lua/entities/gmod_wire_expression2/core/serialization.lua

```diff
@@ -37,6 +37,23 @@ local function logGlonCall( self, glonString, ret, safeGlonObject )
 end
 ]]
 
+-- converts a lua variable's type to a wire typeid
+local function luaTypeToWireTypeid( v )
+	local typename = type( v )
+
+	-- special check for number
+	if typename == "number" then typename = "normal" end
+
+	-- convert full type name to typeid
+	return wire_expression_types[ string.upper( typename ) ][1]
+end
+
+local function glon_not_installed()
```

Comment:
Sorry, I don't have enough faith in your Git skills to think you'd be able to cope with the ensuing repository state :)

(Also, I'm lazy)